Add support for SYLVANIA Wifi LED Smart A19 RGBW Color (2000K)

ryan-ronnander opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Device information
<!--
  Provide and links and additional information about the device you are adding in this PR.
-->
* product_id: `k2nagch3coapc6yd` (Tuya / Sylvania product identifier)
* Model: `40A19FILCCLWIFI`
* SKU: 75807
* Product Description: SYLVANIA Wifi LED Smart A19 RGBW Color and Amber Finish Filament Light Bulb, 7W, for Alexa / Google Assistant, 2000K, No Hub Required, Clear - 1 Pk
* Product URL (LEDVANCE): https://shop.ledvanceus.com/products/sylvania-wifi-led-smart-a19-rgbw-color-amber-finish-filament-light-bulb-7w-2000k-no-hub-required-clear-1pk
* Amazon URL (Amazon): https://www.amazon.com/Sylvania-Filament-Efficient-Assistant-Required/dp/B0BGQ2CCHG

## Home Assistant Device information
<!--
  Provide the Home Assistant device information. This can be found on the device page in HA, on the top left under `Device Info`.
  Please paste that information here.
-->
These bulbs are configured with LocalTuya:
![image](https://github.com/bramstroker/homeassistant-powercalc/assets/61520/34bcb204-2e27-430d-9d9b-43ed6f4dadfa)

## Checklist
<!--
  Please check all the boxes when applicable.
-->

- [X] I have created a single PR per device. When you have multiple submissions please create separate PR's.
- [X] For lights - I have only included the gzipped files (*.gz), not the raw CSV files.
- [X] For lights - I have provided a CSV file per supported color mode. Look that up in Developer Tools -> States

## Additional info
<!--
  Add any additional info we must know about the measurements here.
-->
Measurements were taken with a Shelly PlugUS connected to a lamp with one bulb (I only own one of these). A dummy load was not used due to small fluctuations in wattage over time from the dummy load.

ryan-ronnander commented:
I ended up realizing after the initial pull request that I've had this light bulb misconfigured in HA since originally adding it.

Despite having a static color temperature of 2000K, I still have to configure this particular light with a color temperature range of 2000-2004 as well as set the color temp to false in local tuya:

![image](https://github.com/bramstroker/homeassistant-powercalc/assets/61520/d0c056a9-1b35-422f-8670-b736c9dcab2b)

Anyway, after configuring this bulb correctly I was able to make a color_temp measurement and the power numbers look great in HA.

bramstroker commented:
Nice thanks! also need to provide device information for this one, see other pr #2018 where you can find that.

ryan-ronnander commented:
See my last [comment](https://github.com/bramstroker/homeassistant-powercalc/pull/2018#issuecomment-1918179582) in #2018. 

The true `product_id` is `k2nagch3coapc6yd` which can be used to identify these bulbs. This is obtained using a tool called [tuya-uncover](https://github.com/blakadder/tuya-uncover). 

Output from tuya-uncover: 

```json
"A19 RGBA": {
    "id": "eba6e1a599247d77bcqksy",
    "local_key": "J6N^09b2-F!+^Je(",
    "category": "dj",
    "uuid": "3c3d92e2e4486488",
    "product_id": "k2nagch3coapc6yd",
    "dps": {
        "20": false,
        "21": "white",
        "210": "AAAA",
        "22": 130,
        "24": "00de03e8023d",
        "25": "000e0d0000000000000000c803e8",
        "26": 0,
        "27": "",
        "28": "",
        "31": "AAA=",
        "32": "AAA=",
        "33": "AAEA3gPoAJYAlgPo",
        "34": false,
        "39": 0,
        "41": true
    }
},
```

Device information in HA (Local Tuya or Tuya Local):

![image](https://github.com/bramstroker/homeassistant-powercalc/assets/61520/1cf67f6a-5c7d-4246-ba3b-22e7d971dc90)

Again, the device info merely shows the unique `id`, similar to a mac address.

bramstroker commented:
See my comment on other PR.
Preferrered to have the product_id added as an alias in model.json and keep the directory name untouched.

ryan-ronnander commented:
Good news, I must have been looking at these in the wrong light originally, but I have found the model number stamped at the base: `40A19FILCCLWIFI`. I've edited my original pull request message.

* Model number added, renamed to device folder to `40A19FILCCLWIFI`. 
* Moved from `ledvance` to newly created `sylvania` folder.
* Added `k2nagch3coapc6yd` alias.

A few other details are mentioned in the other PR.

bramstroker commented:
Thanks a lot for all your efforts getting this done.
